Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -219,20 +219,21 @@ def main() -> None:
         orca_client = OrcaClient(client=client)
         if command == "orca-get-alerts":
             demisto_args = demisto.args()
-            alerts = orca_client.get_alerts_by_filter(alert_type=demisto_args.get('alert_type'),
-                                                      asset_unique_id=demisto_args.get('asset_unique_id'))
-            if isinstance(alerts, str):
-                return_error(alerts)
-            if not alerts:
-                return_error("Alerts not exists")
+            alert_type = demisto_args.get('alert_type')
+            asset_unique_id = demisto_args.get('asset_unique_id')
+            alerts = orca_client.get_alerts_by_filter(alert_type=alert_type, asset_unique_id=asset_unique_id)
+            if not alerts or isinstance(alerts, str):
+                error_message = f"Could not find alerts from {alert_type=}" if alert_type else f"Could not find alerts from {asset_unique_id=}"  # noqa: E501
+                alerts = {"alerts": error_message}  # type: ignore
```

Comment:
why did you add `type: ignore`? 
I think we should return an empty list instead of the error message

Reply:
done